iron-doc-nav.html

```diff
@@ -110,14 +110,24 @@
 
```

Comment:
Won't let me add a comment further up.

Just noticed that on line 60, section is redefined again. Can you change this to use another variable name?